src/preset_cli/cli/superset/sync/native/command.py

```diff
@@ -326,7 +325,7 @@ def get_dataset_filter_uuids(config: AssetConfig) -> Set[str]:
     """
     dataset_uuids = set()
     for filter_config in config["metadata"].get("native_filter_configuration", []):
-        for target in filter_config["targets"]:
+        for target in filter_config.get("targets", dict()):
```

Comment:
Could you replace this with:
``` python
        for target in filter_config.get("targets", {}):
```

I believe it's what's breaking CI and it's aligns with other occurrences in the repo.

Thank you! 🙏